doc/source/guide/installation/linux.rst

```diff
@@ -5,11 +5,16 @@ Linux
 Overview
 --------
 
+.. warning::
```

Comment:
What happened to the windows section (win.rst). Looks like it is gone in master? Was this on purpose?